### PR TITLE
Drink and Drinks class

### PR DESCRIPTION
**Pull request by IfhaamZ**
Completed the two classes - added the counter as well

### CODE REVIEW

File: src/main/java/Main.java
Comment (imalisina) [inline src/main/java/Main.java:1]: Unstage the main class then send another pull request as we don't want to apply changes in Main class to other branches

File: src/main/java/classes/Drinks.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    }
+    
+    /*
+     * Method Name : drinkSeeder()
+     * Parameters : none
+     * Description : - create all drink objects
+     */
+    private void drinkSeeder()
+    {
+        allDrinks.add(new Drink("Saffron Tea", 3.99, 4, Category.HOT, "Bread"));
```
Comment (imalisina) [inline src/main/java/classes/Drinks.java:28]: what is the Bread value ?
I saw that also in Foods, if it is default value, change it to an Empty string

File: src/main/java/classes/Drinks.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+     * Description : display all drinks that are in the HOT drinks category
+     */
+    public void displayHotDrinks()
+    {
+        System.out.println("Hot Drinks Menu:\n");
+
+        for (int i = 0; i < allDrinks.size(); i++) {
+            if (allDrinks.get(i).category == Category.HOT)
+                System.out.println(
+                        "[" + (i + 1) + "] " + allDrinks.get(i).toString() + "\n");
```
Comment (imalisina) [inline src/main/java/classes/Drinks.java:53]: Send me the tested CLI result

File: src/main/java/Main.java
Comment (IfhaamZ) [reply src/main/java/Main.java:1]: sent it

File: src/main/java/classes/Drinks.java (hunk 1 above)
Comment (IfhaamZ) [reply src/main/java/classes/Drinks.java:28]: Bread is the pairing, its like pita bread

File: src/main/java/classes/Drinks.java (hunk 2 above)
Comment (IfhaamZ) [reply src/main/java/classes/Drinks.java:53]: ok

**Review by imalisina — CHANGES_REQUESTED** [review]
Edit

File: src/main/java/classes/Drinks.java (hunk 1 above)
Comment (imalisina) [reply src/main/java/classes/Drinks.java:28]: Aha, then it is fine

**Review by imalisina — APPROVED** [review]
Approved
